blockstore/settings/base.py

```diff
@@ -96,6 +96,7 @@ def root(*x):
             # the correct unicode implementation.
             # Note that this limits the length of InnoDB indexed columns to 191 characters.
             'charset': 'utf8mb4',
+            'init_command': 'SET NAMES utf8mb4',
```

Comment:
https://dev.mysql.com/doc/refman/8.0/en/set-names.html

I didn't update the comment because it was still reflective of the purpose of the lines written.